x-pack/plugins/enterprise_search/server/routes/enterprise_search/connectors.ts

```diff
@@ -203,9 +204,24 @@ export function registerConnectorRoutes({ router, log }: RouteDependencies) {
       },
     },
     elasticsearchErrorHandler(log, async (context, request, response) => {
-      const { client } = (await context.core).elasticsearch;
-      await startSync(client, request.params.connectorId, SyncJobType.ACCESS_CONTROL);
-      return response.ok();
+      try {
+        const { client } = (await context.core).elasticsearch;
+        await startSync(client, request.params.connectorId, SyncJobType.ACCESS_CONTROL);
+        return response.ok();
+      } catch (error) {
+        if (isAccessControlDisabledException(error)) {
+          return createError({
+            errorCode: ErrorCode.ACCESS_CONTROL_DISABLED,
+            message: i18n.translate('xpack.enterpriseSearch.server.connectors.accessControlSync', {
+              defaultMessage:
+                'Access Control Sync cannot be created. Document Level Security is disabled.',
```

Comment:
@leemthompo does the wording look fine? This may be visible to the use as an error flash message on top of the page.